yastd/dbus_api.md

```diff
@@ -3,27 +3,167 @@
 
 Iface: o.o.YaST.Installer1.Language
 
-stupid api:
-  AvailableLanguages -> array(string)
+methods:
 
-minimal working api:
-  needed: human readable name (how localized?), identifier
+  ToInstall(array(string)) -> void
```

Comment:
An important thing about **string**:
Even if it is the implementation type at the D-Bus layer, we should design the API in terms of specialized constant strings. 
Here, a locale identifier as provided by glibc-locale.rpm

Reply:
we, we should be consistent on API side. so if we use here identifier from glibc-locale, we should use same identifier in list of available languages